x-pack/metricbeat/modules.d/enterprisesearch-xpack.yml.disabled

```diff
@@ -1,11 +0,0 @@
-# Module: enterprisesearch
```

Comment:
This file was committed by mistake I think, I see this come in when changing branches occasionally.

Reply:
yep probably changing branches